### ISSUE TITLE
Typescript: import * as arecibo from 'arecibo' fails

### ISSUE THREAD

**Issue by BernhardRode** [nucleode/arecibo#4]
**Describe the bug**
We are using fastify together with typescript in our environment. We would love to use arecibo, but we cannot use:

```typescript
import * as fastify from 'fastify'
import * as arecibo from 'arecibo'

const server = fastify()
server.register(arecibo)
```

and 

We get the following error:

```
[ERROR] 08:09:01 ⨯ Unable to compile TypeScript: 

src/index.ts(26,19):
error TS2345: 
Argument of type 'typeof import("/Users/XXXX/Development/appname/node_modules/arecibo/lib/index")' 
is not assignable to parameter of type 'Plugin<Server, IncomingMessage, ServerResponse, { readinessURL: string; livenessURL: string; }>'.

Type 'typeof import("/Users/XXXX/Development/appname/node_modules/arecibo/lib/index")'

provides no match for the signature 

'(instance: FastifyInstance<Server, IncomingMessage, ServerResponse>, opts: { readinessURL: string; livenessURL: string; }, callback: (err?: FastifyError) => void): void'.
```

When we do:

```typescript
import * as fastify from 'fastify'
const arecibo = require('arecibo')

const server = fastify()
server.register(arecibo)
```

or 

```typescript
import * as fastify from 'fastify'
import * as arecibo from 'arecibo'

const server = fastify()
server.register(arecibo as any)
```

Everything is running fine.

For me it feels like something in the calling signature in fastify 2.0.0 changed, which breaks the plugin.

**To Reproduce**

1. Setup a base project using typescript like https://github.com/fastify/fastify/blob/master/docs/TypeScript.md
2. install arecibo
3. import it like described and start the server

**Expected behavior**

Should result in the same behaviour.

**Comment by fox1t** [comment]
Hi. Thanks for the report. We will look into this today. I think we need to update arecibo to support fastify 2.0.0.

**Comment by fox1t** [comment]
Hi @BernhardRode i just published version 1.0.0 of arecibo, dropping support for fastify 1.x.x and adding new way of exporting. We now support `require()`, `import arecibo from 'arecibo'` and `import * as  arecibo from 'arecibo'`.

Let me know if this version fixes your issue.

MS

**Comment by BernhardRode** [comment]
Thank you very much. It is working like a charm!

You are awesome and just made my day!
